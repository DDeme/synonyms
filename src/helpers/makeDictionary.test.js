const makeDictionary = require('./makeDictionary')

describe('test of make ditionary', () => {
  const exampleDictionary = [
    'big large',
    'large huge',
    'small little',
    'apple banana',
  ]

  test('should return sample prepared dictionary with relations', () => {
    const dictionary = makeDictionary(exampleDictionary)
    expect(dictionary).toMatchSnapshot()
  })
})
