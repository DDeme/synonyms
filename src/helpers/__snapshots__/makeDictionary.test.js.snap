// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test of make ditionary should return sample prepared dictionary with relations 1`] = `
Object {
  "apple": Array [
    "banana",
  ],
  "banana": Array [
    "apple",
  ],
  "big": Array [
    "large",
  ],
  "huge": Array [
    "large",
  ],
  "large": Array [
    "huge",
    "big",
  ],
  "little": Array [
    "small",
  ],
  "small": Array [
    "little",
  ],
}
`;
